mslib/mscolab/file_manager.py

```diff
@@ -399,7 +399,7 @@ def get_all_changes(self, op_id, user, named_version=False):
             'comment': change.comment,
             'version_name': change.version_name,
             'username': change.user.username,
-            'created_at': change.created_at.strftime("%Y-%m-%d, %H:%M:%S")
+            'created_at': change.created_at.strftime("%Y-%m-%d, %H:%M:%S %z")
```

Comment:
Not sure if this is the only what we need to do to fractions used
"%Y-%m-%d, %H:%M:%S.%f %z"

Reply:
not yet. I suggest. Get  the parts which you have now passing the tests. 
Then we can merge this. Afterward work on getting the time.sleep(1) away.